docs/antora/modules/ROOT/pages/configuration/docker.adoc

```diff
@@ -1,346 +1,35 @@
 = Configuration with Docker
-Mona Bärenfänger <mona@lightcurve.io>
+Mona Bärenfänger <mona@lightcurve.io> Muhammad Talha <muhammad.talha@lightcurve.io>
 :description: Describes how to configure Lisk Service with Docker.
 :toc:
 :imagesdir: ../assets/images
 :page-previous: /lisk-service/setup/docker.html
 :page-previous-title: Installation with Docker
 :page-next: /lisk-service/management/docker.html
 :page-next-title: Docker commands
-
 :url_references_config: configuration/index.adoc
+:url_github_env: https://github.com/LiskHQ/lisk-service/blob/development/docker/example.env
+:url_docker_commands: management/docker.adoc
+
+On this page, we discuss how to customize the docker installation of Lisk Service.
 
-How to configure Lisk Service.
 
 == Configuring Lisk Service
+The Docker installation of Lisk Service takes all of its configuration from an `example.env` file, which is located in the `lisk-service` directory.
```

Comment:
This statement is incorrect. The `docker/example.env` file is only for reference and specifies all the available configurable environment variables and already specifies the default values.

For the user to use it, they'll need to copy this into a `.env` file and place it next to the `docker-compose.yaml` file in the root directory of Lisk Service.